502.html

```diff
@@ -2,10 +2,10 @@
 <html>
 <title>Ow! -- reddit.com</title>
 <style>
-body{text-align:center;position:absolute;top:50%;margin:0;margin-top:-340px;width:100%}
-h2,h3{color:#222;font:bold 200%/100px sans-serif;margin:0}
+body{text-align:center;position:absolute;top:50%;margin:0;margin-top:-275px;width:100%}
+h2,h3{color:#555;font:bold 200%/100px sans-serif;margin:0}
 h3{color:#777;font:normal 150% sans-serif}
 </style>
-<img src=//www.redditstatic.com/youbrokeit1.png alt="">
-<h2>we took too long to make this page for you</h2>
-<h3>try again and hopefully we will be fast enough this time.
+<img src=//www.redditstatic.com/trouble-afoot.png alt="">
+<h2>sorry, something broke on our end</h2>
+<h3>please try again in a minute.</h3>
```

Comment:
minor request: the `.` in the second sentence on these error pages looks very very strange to me for some reason and i regret adding the original one. could it die please? :)